src/external/external.js

```diff
@@ -0,0 +1,4 @@
+global.choose = function(className) {
```

Comment:
```suggestion
global.choose = function (className) {
```

Reply:
Hmm, my gut feeling is that this logic belongs in Mjølner's `lib/objc.js`, as Mjølner should provide all of the core runtime bits that scripts expect. (Except for module-loading, which is provided by `src/Agent/`.) What do you think?

Reply:
Totally agree, will just leave in this PR update to 14.2.14, cypress test and fix in Execute.cpp.